germanium/django/runner.py

```diff
@@ -0,0 +1,29 @@
+from django.test.runner import DiscoverRunner
+
+from .utils import setup_databases
+
+
+class GermaniumRunnerMixin:
+
+
+    def __init__(self, **kwargs):
+        self.refreshdb = kwargs.pop('refreshdb', False)
+        super().__init__(**kwargs)
+
+    def setup_databases(self, **kwargs):
+        return setup_databases(
+            self.verbosity, self.interactive, self.keepdb and not self.refreshdb, self.debug_sql,
```

Comment:
This will not work if I want to delete existing DB, create new, but keep it, or?